icons/eye-off.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M9.88 9.88a3 3 0 1 0 4.24 4.24" />
-  <path d="M10.73 5.08A10.43 10.43 0 0 1 12 5c7 0 10 7 10 7a13.16 13.16 0 0 1-1.67 2.68" />
-  <path d="M6.61 6.61A13.526 13.526 0 0 0 2 12s3 7 10 7a9.74 9.74 0 0 0 5.39-1.61" />
-  <line x1="2" x2="22" y1="2" y2="22" />
+  <path d="M10.733 5.076a10.744 10.744 0 0 1 11.205 6.575 1 1 0 0 1 0 .696 10.747 10.747 0 0 1-1.444 2.49" />
+  <path d="M14.121 14.121A3 3 0 0 1 9 12a3 3 0 0 1 .879-2.121" />
```

Comment:
```suggestion
  <path d="M14.084 14.158a3 3 0 0 1-4.242-4.242" />
```

Reply:
Oh, indeed, I wonder how I missed that.

But this is also something for you to look into in Lucide Studio. 😅